src/platform/CMakeLists.txt

```diff
@@ -54,8 +54,8 @@ endif()
 if(QUIC_TLS STREQUAL "openssl")
     add_dependencies(platform OpenSSL)
     target_link_libraries(platform
-        ${QUIC_BUILD_DIR}/openssl/lib/libssl.a
-        ${QUIC_BUILD_DIR}/openssl/lib/libcrypto.a)
+        ${QUIC_BUILD_DIR}/openssl/lib/libssl.so
+        ${QUIC_BUILD_DIR}/openssl/lib/libcrypto.so)
```

Comment:
We want to statically link, not dynamically, to openssl.

Reply:
Ok, well then I need to change the root CMakeLists.txt. It outputs .so files.

Reply:
Switched

Reply:
Ok. Looks good. Now things work with `-GNinja`?